.zshrc

```diff
@@ -209,6 +209,7 @@ alias grb2="git rebase -i HEAD~2"
 alias grb3="git rebase -i HEAD~3"
 alias grb5="git rebase -i HEAD~5"
 alias grbt="git rebase -i HEAD~10"
+alias grbtt="git rebase -i HEAD~1010"
```

Comment:
```suggestion
alias grbtt="git rebase -i HEAD~$(echo "obase=2;10" | bc)
```

Reply:
Hardcoding values isn't enterprise-y enough.

Reply:
I measured this across 10000 runs and your change really drags down the perf. Is this a strict requirement?

Reply:
Are you the enterprise version of zsh? If not it will rate limit rc file execution. We only support "zsh for business" subscriptions.

Reply:
I agree with @benbp

Reply:
@maorleger - can you clarify?